src/legacy/ui/public/vis/map/service_settings.js

```diff
@@ -31,7 +32,7 @@ const markdownIt = new MarkdownIt({
 
 const TMS_IN_YML_ID = 'TMS in config/kibana.yml';
 
-uiModules.get('kibana')
+uiModules.get('kibana', ['ngSanitize'])
```

Comment:
:information_source: Apparently the `ngSanitize` plugin was dependent on some weird implicit way earlier to this PR, so we now need to explicitly require it here were we use the `$sanitize` service.